nix/overlays/haskell-packages.nix

```diff
@@ -34,6 +34,10 @@ let
       # - If the library fails its test suite (usually when it runs IO tests), wrap the expression with `lib.dontCheck ()`
       # - <subpath> is usually "."
       # - When adding a new library version here, postgrest.cabal and stack.yaml must also be updated
+      #
+      # To try a modified library locally (for development purposes only), see this gist as example:
+      #   https://gist.github.com/laurenceisla/de88d975b894074c989feffa0c5611cc
+      #
```

Comment:
Ah, I get it. Only production ready examples should be used there.

OK, I'll be adding a new section to that README.

Reply:
@wolfgangwalther @laurenceisla For [Locality of Behaviour](https://htmx.org/essays/locality-of-behaviour/), how about keeping this on the `nix/overlays/haskell-packages.nix`?

It'll be much better if anyone can just read the file and then find the way..

Reply:
> However: Instead of me telling you "you're doing it wrong" all the time ;)... 

I like being told I'm wrong! :smile: 

> maybe we can add a hint with this kind of explanation to the overlay file?

Sounds good!

Reply:
> maybe we can add a hint with this kind of explanation to the overlay file?

Did that, not sure if more info is needed here. Added a mention to the Nix README too, for local packages.